### DISCUSSION TITLE
How to add a member?

### DISCUSSION THREAD

**Discussion by psommerfeld** [makeplane/plane#6204] [Q&A] [answered]
I'm trying to follow https://docs.plane.so/core-concepts/workspaces/members to add a member. But what I see is in the pic attached.
I have no opp'ty to add an email address for a member like I see in the docs, but only to somehow search for a member from a non-existent drop-down list that I don't know how to create.
TIA!
<img width="1039" alt="Screenshot 2024-12-14 at 4 14 38 PM" src="https://github.com/user-attachments/assets/8da708f3-bd92-4502-8640-07f44f5447d2" />

**Comment by psommerfeld** [comment]
Does anyone track issues here?

**Reply by marcungeschikts** [reply, marked as answer]
> Does anyone track issues here?

Not the best way to ask for help :wink:

---
Anyway, looking at the screenshot you attached, it seems you are trying to add a member to a project.

For that, you need to add the member to the workspace first.

From there, (**Workspace Settings > Members** — you need to be a workspace admin), you will be able to add the member using their email.

**Reply by psommerfeld** [reply]
Thanks, where is best way / place?
I looked for a Reddit group, but didn't know where else would be appropriate.

**Reply by marcungeschikts** [reply]
> Thanks, where is best way / place? I looked for a Reddit group, but didn't know where else would be appropriate.

I didn’t mean to suggest this was the wrong place to ask, but the way it was phrased felt a bit rude. If I came across as rude myself, I apologize. 🙏

That said, you’re right—GitHub Discussions might not be the best place for a user issue like adding a Member via email.

If you’re a new or free user, I’d recommend checking out the [Plane Discord channel](https://discord.com/invite/A92xrEGCge) for community support.

Alternatively, we chose a [Pro license](https://plane.so/pricing), which provides extra features, and **better support**.
It's another way to contribute to an open-source project.

**Comment by psommerfeld** [comment]
Oh ok, I guess I wasn't sure if people actually monitor the discussions. Well maybe I should have worded better.
Thanks for the Discord link. Exactly what I need! I see Discord mentioned on their Github now.

**Comment by vihar** [comment]
@psommerfeld Hi, Vihar here - I'm one of the maintainers at Plane. Just wanted to clarify that we don't typically monitor org discussions. As Marc mentioned, we're most active on:

- Discord
- GitHub issues (on our main repo: github.com/makeplane/plane/issues)
- In-app chat support (for cloud users)

I'm sharing this context to help new users know where to best reach us. Since we're a small team, we've focused our support efforts on these three channels to ensure we can provide the best possible assistance.

And @marcungeschikts - thank you for helping out. 🙏🏻
